peregrine/resources/submission/graphql/node.py

```diff
@@ -389,8 +390,14 @@ def resolve_node(self, info, **args):
         (not a gdcdatamodel Case)).
 
     """
+    q = query_with_args(psqlgraph.Node, args, info)
+    return [__gql_object_classes[n.label](**load_node(n, info, Node.fields_depend_on_columns)) for n in q.all()]
+
 
-    q = get_authorized_query(psqlgraph.Node)
+def query_with_args(cls, args, info):
+    """Sends a query and applies the arguments."""
```

Comment:
could you add some info in the docstring for what `cls`, `args`, and `info` are? (like the example [here](https://github.com/uc-cdis/rfcs/blob/master/text/0003-style-guide.md#documentation))

Reply:
I can, but these 3 are everywhere in this file (I just moved this function, I didn't write it)